clang/test/Driver/systemz-alignment.c

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang --target=s390x-linux -S -emit-llvm -o - %s | FileCheck %s
```

Comment:
Why is this a Driver test rather than a CodeGen test using clang_cc1?

Reply:
I found some other align tests there, like arm-alignment.c, so I thought this should work..?

Reply:
Sorry, I missed this when reviewing.

We generally only use Driver tests when we're trying to test some aspect of the driver.  For arm-alignment.c, it isn't really trying to test the code generation itself; it's trying to test the effect of alignment-related flags.

Reply:
Sorry, my bad. There is a SystemZ test for this already, so I moved these tests into it instead. I created a new PR: https://github.com/llvm/llvm-project/pull/73230.